components/src/NavBar/SubMenu.js

```diff
@@ -76,24 +76,22 @@ BaseSubMenu.defaultProps = {
 
 const SubMenu = styled(BaseSubMenu)(
   ({ renderArrow }) => ({
+    listStyle: "none",
+    margin: "0",
     color: subMenuStyles.nameColor,
-    display: "flex",
-    maxWidth: "20em",
-    flexDirection: "column",
-    fontSize: theme.fontSizes.small,
     backgroundColor: subMenuStyles.backgroundColor,
     borderRadius: theme.radii.medium,
     borderTop: `1px solid ${subMenuStyles.borderColor}`,
     borderBottom: `1px solid ${subMenuStyles.borderColor}`,
-    boxShadow: theme.boxShadows,
+    boxShadow: theme.boxShadows.small,
```

Comment:
Nikola just merged a branch this morning that changes `boxShadows` to `shadows`.

Reply:
rebased and made the fix